lib/r10k/puppetfile.rb

```diff
@@ -88,6 +87,14 @@ def loaded?
     @loaded
   end
 
+  # @param [String] forge
+  def set_forge(forge)
+    unless @forge.nil?
+      raise R10K::Error.new(_('You cannot declare multiple \'forge\' settings in the same Puppetfile'))
+    end
+    @forge = forge
```

Comment:
There was a comment in the previous PR about needing to make sure this is set to the default from the forge module if `allow_overrides` is false. Is that actually necessary? It kind of looks like the logic to just ignore this field in the install action would take care of that case?

Reply:
@Magisus if the part of this commit for this file isn't necessary then yes I believe this wouldn't be needed anymore either.